### PR TITLE
Ability to display article

### PR DESCRIPTION
**Pull request by jyuw**
Pull Request template.

#### Description
Changes proposed in this pull request:
* user can access article page and see title and content

#### What I have learned working on this feature:
* reminding us how to cherry-pick
* how to display an article

### CODE REVIEW

File: features/user_should_be_able_to_create_articles.feature [hunk 1]
```diff
@@ -4,26 +4,26 @@ Feature: User should be able to create articles
...
-    Given I am on the "Create Article" Page
+    Given I am on the "Create Article" page
     And I fill in "Content" with "Dreamteam for the win!"
     Then I click on "Create Article"
     And I should see "Title can't be blank"
 
 
   Scenario: User forgets to fill in Title
-    Given I am on the "Create Article" Page
+    Given I am on the "Create Article" page
```
Comment (magnus-thor) [inline features/user_should_be_able_to_create_articles.feature]: When writing the scenarios follow the Given, when, then convention.
If you need a reminder on it, go over this https://martinfowler.com/bliki/GivenWhenThen.html

**Review by magnus-thor — CHANGES_REQUESTED** [review]
go over the scenarios and follow Given When Then

File: features/user_can_see_articles.feature [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+  Background:
+    Given the following articles exist
+    | title               | body                  |
+    | Holger is the best  | Dreamteam for the win |
+
+  Scenario: User can see articles displayed
+    Given I am on the "Holger is the best" page
+    Then I should see "Holger is the best"
+    And I should see "Dreamteam for the win"
```
Comment (magnus-thor) [inline features/user_can_see_articles.feature]: You forgot this. Good job on the other scenarios.

**Review by magnus-thor — APPROVED** [review]

**Comment by larathordar** [comment]
looks great

**Pull request merged**